File: ClinicSystem/src/com/example/clinicsystem/models/classes/M_Prescription.java
package com.example.clinicsystem.models.classes;

import java.util.List;

public class M_Prescription {
    private String date;
    private List<M_PrescriptionInstruction> instructions;
    private int patientID, dentistID;

    public M_Prescription(String date, List<M_PrescriptionInstruction> instructions, int patientID, int dentistID) {
        this.date = date;
        this.instructions = instructions;
        this.patientID = patientID;
        this.dentistID = dentistID;
    }

    public M_Prescription() {
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public List<M_PrescriptionInstruction> getInstructions() {
        return instructions;
    }

    public void setInstructions(List<M_PrescriptionInstruction> instructions) {
        this.instructions = instructions;
    }

    public int getPatientID() {
        return patientID;
    }

    public void setPatientID(int patientID) {
        this.patientID = patientID;
    }

    public int getDentistID() {
        return dentistID;
    }

    public void setDentistID(int dentistID) {
        this.dentistID = dentistID;
    }
}

File: clinic/M_Prescription.php
<?php
class M_Prescription {
    private $pdate;
    private $instructions;
    private $patientID, $dentistID;

    public function __construct($date, $instructions, $patientID, $dentistID) {
        $this->pdate = $date;
        $this->instructions = $instructions;
        $this->patientID = $patientID;
        $this->dentistID = $dentistID;
    }

    public function getpDate() {
        return $this->pdate;
    }

    public function setDate($pdate) {
        $this->pdate = $pdate;
    }

    public function getInstructions() {
        return $this->instructions;
    }

    public function setInstructions($instructions) {
        $this->instructions = $instructions;
    }

    public function getPatientID() {
        return $this->patientID;
    }

    public function setPatientID($patientID) {
        $this->patientID = $patientID;
    }

    public function getDentistID() {
        return $this->dentistID;
    }

    public function setDentistID($dentistID) {
        $this->dentistID = $dentistID;
    }
}

?>